### DISCUSSION TITLE
有没有办法使用命令行得知rime正在使用什么schema

### DISCUSSION THREAD

**Discussion by bincat233** [fcitx/fcitx5#327] [Q&A]
正在写一个polybar脚本, 需要从脚本里获取输入法的状态

我可以通过 `dbus-send --session --print-reply --dest=org.fcitx.Fcitx5 /controller org.fcitx.Fcitx.Controller1.CurrentInputMethod` 得知Fcitx5正在使用什么输入法, 

不知道有没有类似的方法可以知道 rime 正在使用什么 schema, 也就是获取菜单里的这一部分:

![photo_2021-08-23_07-26-35](https://user-images.githubusercontent.com/20554448/130373364-fa3ed7bf-eea6-49f0-9c17-d5da3a07458d.jpg)

**Comment by bincat233** [comment]
或者说, 有没有办法从命令行获取fcitx5右键菜单的内容

**Reply by wengxt** [reply]
你可以给 fcitx5-rime PR 一个返回 schema 信息的接口

**Comment by xingwozhonghua126** [comment]
建议提升fcitx5-rime与fcitx5-table等级一样，schema就相当于输入法

**Reply by wengxt** [reply]
> 
> 
> 建议提升fcitx5-rime与fcitx5-table等级一样，schema就相当于输入法

rime 自己有切换 schema 的机制，不方便这样统一
